examples/ota-requestor-app/ota-requestor-common/BDXDownloader.cpp

```diff
@@ -69,16 +75,29 @@ void BdxDownloader::HandleTransferSessionOutput(TransferSession::OutputEvent & e
         VerifyOrReturn(err == CHIP_NO_ERROR, ChipLogError(BDX, "SendMessage failed: %" CHIP_ERROR_FORMAT, err.Format()));
         break;
     }
-    case TransferSession::OutputEventType::kAcceptReceived:
+    case TransferSession::OutputEventType::kAcceptReceived: {
+        OTAImageProcessorParams params = {
+            .imageFile = outFilePath,
+            .downloadedBytes = 0,
+            // TODO: Set this to Length field of ReceiveAccept
+            .totalFileBytes = 0,
+        };
+        mImageProcessor->SetOTAImageProcessorParams(params);
```

Comment:
- `.imageFile` is a pointer, so it's non-owning. This causes issues about who "owns" the `imageFile` value.
- `outFilePath` is a Linux implementation detail. It shoudl not appear in common API params usable by all OTA image processors
- This AcceptReceive and the OTAImageProcessor interface does not support or account resumption of downloads